src/__screenshot_tests__/display-media-card-screenshot-test.tsx

```diff
@@ -34,3 +34,16 @@ test('DisplayMediaCard group', async () => {
 
     expect(image).toMatchImageSnapshot();
 });
+
+test('DisplayMediaCard with video', async () => {
+    await openStoryPage({
+        id: 'components-cards-display-media-card--default',
+        args: {background: 'video'},
+    });
+
+    const displayMediaCard = await screen.findByTestId('display-media-card');
+
+    const image = await displayMediaCard.screenshot();
+
+    expect(image).toMatchImageSnapshot();
+});
```

Comment:
what about testing some failing case we encompass? i.e. failing video with a background fallback